lib/pharos/phases/base.rb

```diff
@@ -21,8 +21,7 @@ def exec_script(script, vars = {})
 
       def parse_resource_file(path, vars = {})
```

Comment:
This looks a whole lot like the one in `Pharos::Kube`, which I think is in a wrong place too

Reply:
Hmm, it's a class method. This could just call it with `Pharos::Kube.parse_resource_file` I think?

Reply:
That returns a `Kubeclient::Resource`, which is the wrong thing here.